python/pyspark/sql/connect/dataframe.py

```diff
@@ -1584,8 +1584,16 @@ def __getattr__(self, name: str) -> "Column":
                 error_class="NOT_IMPLEMENTED",
                 message_parameters={"feature": f"{name}()"},
             )
+
+        if name not in self.columns:
```

Comment:
How about use `elif:` here ?

Reply:
yeah but this is matched with the non Spark Connect side. I think it's just fine to leave it matched.

Reply:
I checked the pyspark. As you said, I'm +1 for you comment.